### DISCUSSION TITLE
ayuda de instalacion  de tailwind en VSC

### DISCUSSION THREAD

**Discussion by Joseo300964** [tailwindlabs/tailwindcss#14983] [Help]
Buenas tardes necesito su ayuda, voy a instalara tailwind en mi carpeta abierta en VSC me dice que "no s epeude cargar el archivo por que la ejecucion de scripts esta desabilitada en este sistema"  mil gracias por su ayuda

**Comment by leslyCPay** [comment]
Quizas esto te ayude:
https://www.cdmon.com/es/blog/la-ejecucion-de-scripts-esta-deshabilitada-en-este-sistema-te-contamos-como-actuar
